website/content/en/status/report-2025-04-2025-06/foundation-sta.adoc

```diff
@@ -0,0 +1,104 @@
+
+The detailed scope was co-created with core@, srcmgr@, portmgr@, doceng@.
+
+* Must
+** Improve quality of incoming commits (completed)
+** Pre-merge CI (completed)
+** Environment Metadata (not started)
+** Extend CI to the Ports tree (in progress)
+** CI Threat Model (not started)
+** CI Management Process (in progress)
+** Documentation (not started)
+* Should
+** 3rd-party Interoperability (in progress)
+** Automated analysis in tests (in progress)
+** Test Case Management (not started)
+* Could
+** Granular Debugging (not started)
+
+===== Work Package D: Ports and Packages security improvements
+
+This work package intends to modernize and extend security controls in the FreeBSD Ports and Package Collection by: Migrating from our VuXML Vulnerability Database to OSV or similar contemporary format; developing a package audit backend and server to reliably fetch vulnerability data from global agency databases in any format (JSON - NIST) and produce insight and; improving CI tooling for FreeBSD Ports.
+
+The detailed scope was co-created with core@, portmgr@, pkgmgr@, secteam@.
+
+* Must
+** New Database Format (in progress)
+** Set up 2+ Database Instances (not started)
+** Migrate Data from old to new database  (in progress)
+** Add support for new format in pkg(8) (in progress)
+** Upstream engagement (not started)
+** SBOM on demand (not started)
+** Document how to set up build and test targets (not started)
+** Integrate 3rd party test targets  (not started)
+** Continuous Testing (not started)
+* Could
+** Make CI artifacts available (not started)
+
+===== Work Package E: SBOM improvements
+
+This work package intends to improve existing, and implement new, tooling and processes for FreeBSD Software Bill of Materials (SBOM) by implementing: tooling to roll up the individual provenance data/markers from across the tree into a higher-level view; developing tooling to parse/review/inspect the FreeBSD source tree and produce a comprehensive/holistic report to act as a SBOM for the full software stack and; extending pkg to enable this capability for software installed from ports/packages.
+
+The detailed scope was co-created with core@, portmgr@, pkgmgr@, secteam@, releng@
+
+* Must
+** Evaluate projects/solutions available in the wider ecosystem (in progress)
+** Propose the target solution for SBOM (not started)
+** Produce an SBOM in CI (e.g. weekly builds) (in progress)
+** Produce an SBOM as an artifact as part of the release process (in progress)
+** SBOM artifact on demand (in progress)
+** Roll up existing data (not started)
+** Record and explain decisions made (not started)
+* Could
+** Engage with other similar projects (not started)
+
+Commissioning body: Sovereign Tech Agency
```

Comment:
Sponsoring and commissioning is not exactly the same thing, but I suggest using the `Sponsor:` tag here, for consistency with the other reports. That this sponsoring is the result of a commissioning is made clear within the report body.
This is only a suggestion, feel free to keep `Commissioning body:` if you believe it is better.

```suggestion
Sponsor: Sovereign Tech Agency
```